DomainContent/LoadTest/Trivia/trivia.html

```diff
@@ -0,0 +1,278 @@
+                    margin: 10;
+                    width: 100% - 10px;
+                    font-family: 'Raleway', sans-serif;
+                    color: #cccbcb;
+                    background: linear-gradient(#2b2b2b, #303030);
+                }
+    
+                .top-bar {
+                    height: 90px;
+                    background: linear-gradient(#2b2b2b, #303030);
+                    font-weight: bold;
+                    padding-left: 30px;
+                    padding-right: 30px;
+                    display: flex;
+                    align-items: center;
+                    position: fixed;
+                    width: 480px;
+                    top: 0;
+                    z-index: 1;
+                }
+    
+                .content {
+                    margin-top: 90px;
+                    padding: 30px;
+                }
+    
+                input[type=button] {
+                    font-family: 'Raleway';
+                    font-weight: bold;
+                    font-size: 13px;
+                    text-transform: uppercase;
+                    vertical-align: top;
+                    height: 28px;
+                    min-width: 120px;
+                    padding: 0px 18px;
+                    margin-right: 6px;
+                    border-radius: 5px;
+                    border: none;
+                    color: #fff;
+                    background-color: #000;
+                    background: linear-gradient(#343434 20%, #000 100%);
+                    cursor: pointer;
+                }
+                input[type=button].gray {
+                    color: #121212;
+                    background-color: #949494;
+                    background: linear-gradient(#949494 20%, #cacaca 100%);
+                }
+    
+                input[type=button]:enabled:hover {
+                    background: linear-gradient(#000, #000);
+                    border: none;
+                }
+               
+                input[type=button].gray:enabled:hover {
+                    background: linear-gradient(#fff, #fff);
+                    border: none;
+                }
+    
+                input[type=button]:active {
+                    background: linear-gradient(#343434, #343434);
+                }
+                input[type=button].gray:active {
+                    background: linear-gradient(#afafaf, #afafaf);
+                }
+    
+                input[type=button]:disabled {
+                    color: #252525;
+                    background: linear-gradient(#575757 20%, #252525 100%);
+                }
+    
+                input[type=button][pressed=pressed] {
+                    color: #00b4ef;
+                }
+
+                .correct {
+                    color: #00b4ef;
+                }
+    
+            </style>
+    </head>
+    <body>
+        <h1>Trivia</h1>
+        <input type="button" class="gray" id="begin" value="Let's get started!">  </input>
```

Comment:
<input> in HTML doesn't need a closing tag - that's why GH is flagging all of these. You can remove them here and throughout the file